File: HotDesk_PRAI/TMS_HotDesk_Prai/Backup/Library/Library.Root/Object/Log.vb
Namespace [Object]
    Public Class Log

        Private _ID As Integer = 0
        Public Property ID() As Integer
            Get
                Return _ID
            End Get
            Set(ByVal value As Integer)
                _ID = value
            End Set
        End Property

        Private _keyField As String = String.Empty
        Public Property KeyField() As String
            Get
                Return _keyField
            End Get
            Set(ByVal value As String)
                _keyField = value
            End Set
        End Property

        Private _KeyDesc As String = String.Empty
        Public Property KeyDesc() As String
            Get
                Return _KeyDesc
            End Get
            Set(ByVal value As String)
                _KeyDesc = value
            End Set
        End Property

        Private _keyValue As String = String.Empty
        Public Property KeyValue() As String
            Get
                Return _keyValue
            End Get
            Set(ByVal value As String)
                _keyValue = value
            End Set
        End Property

        Private _fieldname As String = String.Empty
        Public Property FieldName() As String
            Get
                Return _fieldname
            End Get
            Set(ByVal value As String)
                _fieldname = value
            End Set
        End Property

        Private _b4Update As String = String.Empty
        Public Property B4Update() As String
            Get
                Return _b4Update
            End Get
            Set(ByVal value As String)
                _b4Update = value
            End Set
        End Property

        Private _afUpdate As String = String.Empty
        Public Property AFUpdate() As String
            Get
                Return _afUpdate
            End Get
            Set(ByVal value As String)
                _afUpdate = value
            End Set
        End Property

        Private _updatedby As String = String.Empty
        Public Property UpdateBy() As String
            Get
                Return _updatedby
            End Get
            Set(ByVal value As String)
                _updatedby = value
            End Set
        End Property

        Private _updateDate As String = String.Empty
        Public Property UpdatedDate() As String
            Get
                Return _updateDate
            End Get
            Set(ByVal value As String)
                _updateDate = value
            End Set
        End Property
    End Class
End Namespace

File: HotDesk_PRAI/Library/Library.Root/Object/Log.cs
namespace Library.Root.Object
{
    public class Log
    {
        private int _ID = 0;
        public int ID
        {
            get { return _ID; }
            set { _ID = value; }
        }

        private string _keyField = string.Empty;
        public string KeyField
        {
            get { return _keyField; }
            set { _keyField = value; }
        }

        private string _KeyDesc = string.Empty;
        public string KeyDesc
        {
            get { return _KeyDesc; }
            set { _KeyDesc = value; }
        }

        private string _keyValue = string.Empty;
        public string KeyValue
        {
            get { return _keyValue; }
            set { _keyValue = value; }
        }

        private string _fieldname = string.Empty;
        public string FieldName
        {
            get { return _fieldname; }
            set { _fieldname = value; }
        }

        private string _b4Update = string.Empty;
        public string B4Update
        {
            get { return _b4Update; }
            set { _b4Update = value; }
        }

        private string _afUpdate = string.Empty;
        public string AFUpdate
        {
            get { return _afUpdate; }
            set { _afUpdate = value; }
        }

        private string _updatedby = string.Empty;
        public string UpdateBy
        {
            get { return _updatedby; }
            set { _updatedby = value; }
        }

        private string _updateDate = string.Empty;
        public string UpdatedDate
        {
            get { return _updateDate; }
            set { _updateDate = value; }
        }
    }
}
